service/turn.rest/pom.xml

```diff
@@ -69,10 +69,32 @@
         <groupId>org.kaazing</groupId>
         <artifactId>k3po-maven-plugin</artifactId>
       </plugin>
+
+
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-failsafe-plugin</artifactId>
+        <configuration>
```

Comment:
This should be able to be inherited and no config needed.  Regardless i should be *IT.java